test/Draw.rb

```diff
@@ -7,84 +7,84 @@ def setup
   end
 
   def test_affine
-    assert_nothing_raised do
+    expect do
```

Comment:
This probably looks prettier without the `do`?

Reply:
@dlemstra I'm not sure I understand what you're suggesting here. Are you suggesting switching it to parenthesis `()`? For the `raise_error` assertions, we must pass a block so that RSpec can rescue from an error internally.

Or do you mean we should use curly braces `{}`? One common convention in Ruby is that single-line blocks use curlies and multi-line blocks use `do`/`end`. I'm open to exploring alternatives, though I'd argue we should use something that is easy to keep consistent across the codebase, [ideally via Rubocop](https://rubocop.readthedocs.io/en/stable/cops_style/#styleblockdelimiters).

Reply:
I meant that we should probably use curly braces here because it is a single line statement.

Reply:
Ah, I see. That makes sense. Do you mind if we hold off on stylistic changes for now, though? I saw a ton of things I wanted to change as I was going through the tests, so I expect a lot will change in the end anyway.

Reply:
Surd